### PR TITLE
Added Longest Common Subsequence [Java]

### PR DESCRIPTION
**Pull request by Lucifer24**
#### By submitting this pull request I confirm I've read and complied with the below requirements.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

### CODE REVIEW

**Comment by yashLadha** [comment]
Try to follow the coding guidelines Mentioned in the PR

File: longest_common_subsequence/Longest_common_subsequence.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	
+	public static void main(String[] args) {
+		Random rand = new Random();
+		int[] array = new int[100];
+		for(int u = 0; u < 100; u++) {
+			array[u] = rand.nextInt(100);	
+		}
+		System.out.println ("largest sum subarray: " + maxSubArray(array));	
+	}
+}
```
Comment (yashLadha) [inline longest_common_subsequence/Longest_common_subsequence.java]: Add a new blank line

File: longest_common_subsequence/Longest_common_subsequence.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import java.util.*;
```
Comment (yashLadha) [inline longest_common_subsequence/Longest_common_subsequence.java]: Try to use only those imports which are relevant to the code. Don't unnecessary add the compile time.

Comment (yashLadha) [inline longest_common_subsequence/Longest_common_subsequence.java]: Try to add only those imports which are relevant to code. Don't unnecessary add the compile tiime

File: longest_common_subsequence/Longest_common_subsequence.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+public class LargestCommonSubsequence {
+	public static int maxSubArray(int [] a) {
```
Comment (Monal5031) [inline longest_common_subsequence/Longest_common_subsequence.java]: Attach `[]` with type descriptor as `int[]` not `int []`

File: longest_common_subsequence/Longest_common_subsequence.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+				maxInt = 0;
+			}		
+		}
+		return maxTill;
+	}
+	
+	public static void main(String[] args) {
+		Random rand = new Random();
+		int[] array = new int[100];
+		for(int u = 0; u < 100; u++) {
```
Comment (Monal5031) [inline longest_common_subsequence/Longest_common_subsequence.java]: space between for and `(`

File: longest_common_subsequence/Longest_common_subsequence.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import java.util.*;
+
+public class LargestCommonSubsequence {
```
Comment (Monal5031) [inline longest_common_subsequence/Longest_common_subsequence.java]: Please change Largest to Longest.

File: longest_common_subsequence/Longest_common_subsequence.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+		return maxTill;
+	}
+	
+	public static void main(String[] args) {
+		Random rand = new Random();
+		int[] array = new int[100];
+		for(int u = 0; u < 100; u++) {
+			array[u] = rand.nextInt(100);	
+		}
+		System.out.println ("largest sum subarray: " + maxSubArray(array));
```
Comment (Monal5031) [inline longest_common_subsequence/Longest_common_subsequence.java]: same here longest not largest.

**Comment by singhpratyush** [comment]
@Monal5031: Why was this closed?

**Comment by Monal5031** [comment]
@singhpratyush He had added the code in different folder and also implemented different algorithm than intended. Also, the algorithm he intended to implement is already implemented.

**Comment by singhpratyush** [comment]
@Monal5031: Alright.

@Lucifer24: I hope you got it. If you wish to contribute using this algorithm, you may do it in some other language.

**Pull request closed without merging**